tasks/Ubuntu.yml

```diff
@@ -1,4 +1,15 @@
 ---
+- name: Check if the repository is available
+  uri:
+    url: "{{ openio_repository_mirror_url_base }}/{{ repo.key }}/{{ repo.value.release }}/{{ ansible_lsb.id | default(ansible_distribution)}}/{{ ansible_distribution_release }}"
+  with_dict: "{{ openio_repository_products }}"
+  when:
+    - repo.value.state is not defined or repo.value.state != 'absent'
+    - openio_repository_check_reachability
+  loop_control:
+    loop_var: repo
+#  no_log: '{{ openio_repository_no_log }}'
```

Comment:
Why is this one commented ?

Reply:
For testing purposes, thanks it's fixed!